### ISSUE TITLE
 Wrong Italian translation, singular / plural is missing

### ISSUE THREAD

**Issue by DjTribunus** [woocommerce/woocommerce#29817]
Hi, I have checked the translations page of the plugin and there is an error with the stock in front-end view.

It is formally correct to say in English:

Only 2 left in stock
or
Only 1 left in stock

In Italian language:

Solo 2 pezzi disponibili
or
Solo 1 pezz**I** disponibil**I**

But in reality when the products are less than 2 the correct translation in Italian is:
Solo 1 pezz**O** disponibil**E**

[Link Woocomerce Translation](https://translate.wordpress.org/projects/wp-plugins/woocommerce/stable/it/default/?filters%5Bterm%5D=pezzi+disponibili&filters%5Bterm_scope%5D=scope_translations&filters%5Bstatus%5D=current&filters%5Buser_login%5D=&filter=Apply+Filters&sort%5Bby%5D=priority&sort%5Bhow%5D=desc)

In conclusion of this string there is also a need for the plural (for the Italian language) but it seems to me that it is not present, I hope I have explained myself correctly.

I am well aware of the possibility of translating this string independently, but again, woocomerce does not provide for the possibility of plural translation of this string (as it should be in the Italian language) but only in the singular, how can you solve this?

`$ display = sprintf (__ ('Only% s left in stock', 'woocommerce'), wc_format_stock_quantity_for_display ($ stock_amount, $ product));`

Thanks

**Comment by tammullen** [comment]
Hi @DjTribunus 

While our goal is to address all the issues reported in this repository, GitHub should be treated as a place to report WooCommerce bugs only. Translation is managed by teams over at translate.wordpress.org
​
The issue you are having seems to be caused by the Italian language translation file so would need to be fixed on https://translate.wordpress.org/projects/wp-plugins/woocommerce/

Please contact the translation team https://translate.wordpress.org/projects/wp-plugins/woocommerce/ for further help.
